trojsten/rules/management/commands/ksp_update_levels.py

```diff
@@ -50,16 +50,16 @@ def handle(self, *args, **options):
             )
 
             # Semester that was running in the same time as the camp.
+            # If no semester was running during the camp, the last running semester at the time.
             # For this semester the levels remain unchanged.
             last_semester_before_level_up = rounds_with_semesters[0].semester
             # The semester from which the participants were invited.
-            associated_semester = rounds_with_semesters[1].semester
+            associated_semester = camp.semester
 
             new_rules_start = timezone.datetime(
                 year=2017, month=9, day=1, tzinfo=timezone.get_default_timezone()
             )
-            associated_semester_from_old_rules = rounds_with_semesters[1].end_time < new_rules_start
-            if associated_semester_from_old_rules:
+            if associated_semester.round_set.first().end_time < new_rules_start:
```

Comment:
Nevadi nam ze sa to zrube ak nie su kola v semestri?

Reply:
Tak, semester, z ktorého ľudia idú na camp musí mať round.. Áno, necheckuje sa to, ale prakticky to tak musí byť. Môžem ale hodiť if